docs/source/compile/quantization.rst

```diff
@@ -0,0 +1,147 @@
+(prototype) Quantization with PT2
```

Comment:
One thing I'm not super clear on is what dtypes are actually supported? Is it CPU? GPU? Both?